saleor/webhook/transport/shipping.py

```diff
@@ -209,36 +142,27 @@ def get_excluded_shipping_data(
 
 def get_excluded_shipping_methods_from_response(
     response_data: dict,
-) -> list[dict]:
+) -> list[ExcludedShippingMethodSchema]:
     excluded_methods = []
     for method_data in response_data.get("excluded_methods", []):
```

Comment:
as in previous comment, this part "leaks" validation - nested part is checked in pydantic, but wrapping key here